.github/workflows/docker-image.yml

```diff
@@ -1,9 +1,9 @@
 name: Docker Image CI
 
 on:
-  push:
-    paths:
-      - 'Dockerfile'
+  # push:
+  #   paths:
+  #     - 'Dockerfile'
```

Comment:
Just checking did you mean to comment this out?

Reply:
I did for now. It was trying to run in my branch, so we will need to modify it for proper automation later.